src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -1059,7 +1061,11 @@ export function submitAppointmentOrRequest(history) {
           ...additionalEventData,
         });
         resetDataLayer();
-        history.push('/new-appointment/confirmation');
+        if (isFeatureHomepageRefresh) {
+          history.push(`/requests/${requestData.id}?confirmMsg=true`);
```

Comment:
Would be good if we had a unit test that checked that after submitting on the review page, we redirect to the right url.